frontend/src/components/card/ProblemCard.tsx

```diff
@@ -1,50 +1,66 @@
 import React from 'react';
 import styled from 'styled-components';
 import { Problem } from '../../api/Problem';
-import { Text } from '../core/Text';
-import {
-  CenteredContainer, FlexHorizontalContainer, FlexLeft, FlexRight,
-} from '../core/Container';
+import { LargeText, SelectedItemText, Text } from '../core/Text';
+import { getDifficultyDisplayButton } from '../core/Button';
+import { SelectedItemContainer } from '../core/Container';
+import { DivLink } from '../core/Link';
 
 type ProblemCardProps = {
   problem: Problem,
-  onClick: (problemId: string) => void,
 };
 
 const Content = styled.div`
   display: block;
-  margin: 10px;
-  border-radius: 5px;
-  box-shadow: 0 -1px 4px rgba(0, 0, 0, 0.12);
+  margin: 15px 10px;
+  border-radius: 12px;
+  box-shadow: 0 2px 8px rgba(0, 0, 0, 0.12);
   background-color: ${({ theme }) => theme.colors.white};
+  text-align: left;
   
   &:hover {
     cursor: pointer;
   }
 `;
 
+const InnerContent = styled.div`
+  padding: 15px 30px;
+`;
+
+const ProblemTagContainer = styled(SelectedItemContainer)`
+  margin: 5px 10px 5px 0;
+  box-shadow: 0 1px 8px rgba(0, 0, 0, 0.12);
+  border-radius: 8px;
+`;
+
+const ProblemTagText = styled(SelectedItemText)`
+  font-weight: normal;
+`;
+
+const TitleText = styled(LargeText)`
+  margin: 10px 12px 10px 0;
+  display: inline;
+  vertical-align: middle;
+`;
+
 function ProblemCard(props: ProblemCardProps) {
-  const { problem, onClick } = props;
+  const { problem } = props;
 
   return (
-    <Content onClick={() => onClick(problem.problemId)}>
-      <FlexHorizontalContainer>
-        <FlexLeft>
-          <CenteredContainer>
-            <Text bold>{problem.name}</Text>
-            <br />
-            <Text>{problem.description.substring(0, 50)}</Text>
-          </CenteredContainer>
-        </FlexLeft>
-
-        <FlexRight>
-          <CenteredContainer>
-            <Text>{problem.approval ? 'Approved' : 'Pending Approval'}</Text>
-            <br />
-            <Text>{problem.difficulty}</Text>
-          </CenteredContainer>
-        </FlexRight>
-      </FlexHorizontalContainer>
+    <Content>
+      <DivLink to={`/problem/${problem.problemId}`}>
+        <InnerContent>
+          <TitleText>{problem.name}</TitleText>
+          {getDifficultyDisplayButton(problem.difficulty, true)}
+          <Text>{`${problem.description.substring(0, 80)}...`}</Text>
```

Comment:
Note: I changed all of the problems under the support@codejoust.co account to remove the "Description" header in the description.